plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -852,6 +903,22 @@ function MosaicMenu:_recalculateDimen()
             height = corner_mark_size,
         }
     end
+
+    -- Create or replace progress_widget if needed
+    local progress_bar_width =  self.item_width * 0.60;
+
```

Comment:
While I have no special preference to do without the blank line here, removing as per @poire-z's suggestion before merging. ^_^
```suggestion
```